### PR TITLE
Revamp the I/O BBEs

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/573#issuecomment-755994933

## Test environment
- SLP9
- Java 11
- macOS

### CODE REVIEW

File: examples/xml-io/xml_io.out [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # To run this sample, navigate to the directory that contains the
-# `.bal` file, and execute the `bal run` command below.
+# `.bal` file, and execute the `ballerina run` command below.
```
Comment (niveathika) [inline examples/xml-io/xml_io.out:2]:
```suggestion
# `.bal` file, and execute the `bal run` command below.
```

Comment (BuddhiWathsala) [reply examples/xml-io/xml_io.out]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1174/commits/6361926fb645bcaed2658e294c8e1402ec57d577

File: examples/byte-io/byte_io.bal [hunk 2]
```diff
@@ -1,63 +1,23 @@
...
-        public function close() returns error?;
-    } channelResult = ch;
-    var cr = channelResult.close();
-    if (cr is error) {
-        log:printError("Error occurred while closing the channel: ", err=cr);
-    }
-}
+    // Read the file content as a byte array using the given file path.
+    byte[] bytes = check io:fileReadBytes(imagePath);
+    // Write the content that already read to the given destination file.
```
Comment (praneesha) [inline examples/byte-io/byte_io.bal:11]:
```suggestion
    // Write the already-read content to the given destination file.
```

Comment (BuddhiWathsala) [reply examples/byte-io/byte_io.bal]: Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/1174/commits/21133f9834711318c2d2e96727ebe4e161ce7caf

**Pull request merged**